File: Python/Igual ou diferente.py
num1 = int (input())
num2 = int (input())
num3 = int (input())
if num1==num2==num3:
    print(1)
elif num1==num2 or num1==num3 or (num2==num3):
    print(3)
else:
    print(2)

File: C++/Igual ou diferente.cpp
/*
    Autor : Sandeison Silva Fernandes
    Data: 02 / 06 / 2015

*/

#include <iostream>
using namespace std;

int main ( )
{
    int num_1, num_2,num_3;

    cin >> num_1 >> num_2 >> num_3;
    
    if ((num_1 == num_2)&&(num_1==num_3)){
        cout <<"1"<< endl;        
    }else{
        if (( num_1 != num_2 )&&( num_1!=num_3 )&&( num_2!=num_3 )){
            cout <<"2"<< endl;
        }else{
            cout <<"3"<< endl;
        }


    }

    return 0 ;
}
